Header header
float64 pan_angle
float64 tilt_angle
